server/routerlicious/pnpm-lock.yaml

```diff
@@ -3887,9 +4406,25 @@ packages:
       - webpack-cli
     dev: true
 
+  /@fluidframework/common-definitions@0.20.1:
+    resolution: {integrity: sha512-KaoQ7w2MDH5OeRKVatL5yVOCFg+9wD6bLSLFh1/TV1EZM46l49iBqO7UVjUtPE6BIm0jvvOzJXULGVSpzokX3g==}
+    dev: false
+
   /@fluidframework/common-definitions@1.1.0:
     resolution: {integrity: sha512-WQYtG9tkx2j7i1JSXPvwLnQsqCOZAghMj0aGciqjZVNppUly/XBpAjb4V6FEUCEjxCScPKhyE+1rhV1ep52NgA==}
 
+  /@fluidframework/common-utils@1.1.1:
```

Comment:
Not necessary for this experiment, but when we do this for real (I'll optimistically avoid "if/when" 😄) we should bring gitrest and historian to common-utils 3.1.0 to match routerlicious, and I guess bring syncpack in to server.